activerecord/test/cases/migration/command_recorder_test.rb

```diff
@@ -281,17 +281,47 @@ def test_invert_add_foreign_key
         assert_equal [:remove_foreign_key, [:dogs, :people]], enable
       end
 
+      def test_invert_remove_foreign_key
+        enable = @recorder.inverse_of :remove_foreign_key, [:dogs, :people]
+        assert_equal [:add_foreign_key, [:dogs, :people]], enable
+      end
+
```

Comment:
Alright, did the thing. Good catch, I'd forgotten to even write that functionality.